chamber/models/__init__.py

```diff
@@ -230,13 +243,15 @@ def _save(self, is_cleaned_pre_save=None, is_cleaned_post_save=None, force_inser
         kwargs.update(self._get_save_extra_kwargs())
 
         self._pre_save(change, self.changed_fields, *args, **kwargs)
+        self._call_dispatcher_group('pre_save_dispatchers', self.changed_fields, *args, **kwargs)
```

Comment:
Volám to hned za _pre_save nebo _post_save metodě. Může se hodit moci specifikovat, jestli se to má odpálit při pre sabe nebo post save.